### ISSUE TITLE
Broken HTML encoding/escaping in notification mails

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#6343]
### Expected behavior
When I get notification mails about received introductions the encoding should be ok.

### Actual behavior
When I get notification mails about received introductions the encoding is broken.

### Steps to reproduce the problem
I've configured notification mails to only use 'plain text' in my user settings (text-only notification emails). When I get a notification about a new introduction this looks like this:

````
You&#039;ve received an introduction from &#039;Some-User&#039; at LIBRANET.de

Please visit https://libranet.de to approve or reject the introduction.
https://libranet.de/notify/view/162677

Thank You,
Steffen K9, LIBRANET.de Administrator
````

### Friendica version you encountered the problem
2019.01-rc (no, it's 2018.12-rc) ;-)

### Friendica source (git, zip)
git

### PHP version
7.2.10

### SQL version
10.1.34

**Comment by AlfredSK** [comment]
Have you seen the Friendica version in my report? :-P

**Comment by MrPetovan** [comment]
Ha ha ha, touché.

**Comment by MrPetovan** [comment]
Closed via #6353
